pages/robots/rbts-takehome-device-setup.md

```diff
@@ -1,43 +1,65 @@
 # Android Device Setup
 
-Follow these steps to install and run the **Take Home** application on an Android device:
- 
-## Part 1: Setting up Your Device
+Follow these steps to install and run the **Take Home** application on an Android device.
 
-* __Go to the 'Build number' section.__
+## Step 1: Setting up Your Device
 
-Here’s how to get there on a few popular devices:
+Here we will show you how to activate "Developer Options".
 
-Stock Android: Settings > About phone > Build number
+* __Locate 'Build number' on your device.__
 
-Samsung Galaxy S5: Settings > About device > Build number
+Follow these directories to find 'Build number' on a few popular devices.
 
-LG G3: Settings > About phone > Software information > Build number
+    Stock Android: Settings > About phone > Build number
+    Samsung Galaxy S5: Settings > About device > Build number
+    LG G3: Settings > About phone > Software information > Build number
+    HTC One (M8): Settings > About > Software information > More > Build number
 
-HTC One (M8): Settings > About > Software information > More > Build number
+![BuildNumber](images/rbts-device-build-number.jpg)
```

Comment:
@Jackbui96 Can you also add space between words? "BuildNumber" will become `alt="BuildNumber"` in HTML and this is not so good in terms of accessibility. See more at https://ux.shopify.com/writing-for-all-people-how-to-use-alternative-text-well-1205a18307a1

Reply:
@xyb994 Thank you for pointing that out to me. I did not know the importance of Alternative Text before reading the article that you sent me, now I know why spacing and clarification is important. I also finished updating the Alternative Text, please review.